src/Options/Form.php

```diff
@@ -157,6 +157,8 @@ public function write(): void
 								} else if (file_exists($opt['path'])) {
 									/** @noinspection PhpIncludeInspection */
 									include $opt['path'];
+								} else if (!empty($opt['fn']) && !is_string($opt['fn']) && is_callable($opt['fn'])) {
```

Comment:
Лучше назвать ключ `render`, чтобы было понятно за что он отвечает. Так же достаточно проверить, что ключ установлен и значение можно вызвать как функцию

```suggestion
								} else if (isset($opt['render']) && is_callable($opt['render'])) {
```

Reply:
Подразумевалось, что в _fn_ будет передана исключительно функция, а не строковой адрес на нее

Reply:
Например
```php
$COMPOSER_MODULE->options->addOption('COMPOSER_HTML', [
	'tab' => 'default',
	'type' => 'html',
	'fn' => function() {
		global $COMPOSER_PATH, $COMPOSER_PATH_default;
		// Получить обновленное значение после сохранения
		$COMPOSER_PATH = Option::get(ADMIN_MODULE_NAME, 'COMPOSER_PATH', $COMPOSER_PATH_default);
		echo ComposerUI::instance()->render();
	}
]);
```

Reply:
Это не так важно в каком формате будет передана функция. Главное чтобы это действительно была функция и ее можно было вызвать. Это может быть и Closure, как в твоем случае, так и строка с именнем функции, так метод класса/объекта в формате массива. Подробнее тут, [что php считает callable](https://www.php.net/manual/ru/language.types.callable.php)